src/Elementor/Widgets/CommentList/partials/html5-comment.php

```diff
@@ -11,7 +11,7 @@
             </time>
         </div><!-- .comment-metadata -->
 
-        <?php if ( ! is_admin() ) {
+        <?php if ( !is_admin() ) {
```

Comment:
Please use the [Alternative syntax for control structures](https://www.php.net/manual/en/control-structures.alternative-syntax.php#control-structures.alternative-syntax) in template files.

Reply:
Done